src/fpm/cmd/new.f90

```diff
@@ -103,6 +104,11 @@ subroutine cmd_new(settings)
 
     ! create NAME/README.md
     call warnwrite(join_path(settings%name, 'README.md'), littlefile)
+    
+    gitignorefile=[character(len=5) :: 'build']
```

Comment:
Should we only ignore the build directory in the top-level?

```suggestion
    gitignorefile=[character(len=5) :: '/build/']
```

Having a directory named build seems like something we don't want to globally disallow. The currently proposed solution would for example ignore the directory [`src/tomlf/build`](https://github.com/toml-f/toml-f/tree/main/src/tomlf/build) in the TOML Fortran source tree.

Reply:
To me, `build` folders are generally excluded from VC because their contents can always be generated from the source code. Depending on the platform/architecture the build was ran on, it might also not work on another machine. But not quite sure about `src/toml/build`, though. 🤔